### DISCUSSION TITLE
python 2 compatibility

### DISCUSSION THREAD

**Discussion by pgajdos** [boto/botocore#2777] [Q&A] [answered]
I would like to clarify whether you still want to support python 2. `setup.py` claims the module is python 3 only, however the code still depends on `six`.

**Comment by nateprewitt** [comment, marked as answer]
Hi @pgajdos, the short answer is we do not support Python 2 as stated in the README and trove classifiers.

botocore doesn't depended on six as an external dependency either. We vendor a copy which we intend to keep in place for the time being. This is because we've exposed some of it through our `compat` module which users have used in their own projects. Removing it now is likely to cause unnecessary breakages.

**Comment by pgajdos** [comment]
Thank you for clarification.
